server/src/main/java/org/apache/gravitino/server/web/rest/TableOperations.java

```diff
@@ -140,11 +164,21 @@ public Response createTable(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "load-table." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "load-table", absolute = true)
+  @AuthorizationExpression(
+      expression =
+          "((ANY(USE_CATALOG,METALAKE,CATALOG)) && "
+              + "(SCHEMA::OWNER || ((ANY(USE_SCHEMA,METALAKE,CATALOG,SCHEMA)) &&"
+              + " ( ANY(SELECT_TABLE,METALAKE,CATALOG,SCHEMA,TABLE) || "
```

Comment:
missing ANY(CREATE_TABLE,METALAKE,CATALOG,SCHEMA,TABLE)?

Reply:
Previously discussed with @jerqi , having the CREATE_TABLE permission does not imply having the LoadTable permission. This is documented in the issue: [https://github.com/apache/gravitino/issues/6789](https://github.com/apache/gravitino/issues/6789)

Reply:
Another idea is for me to develop an IDEA plugin to implement keyword highlighting and add rainbow parentheses for the expressions.